lib/utils.js

```diff
@@ -78,7 +78,11 @@ function filterHeaders (headers, filter) {
 
 function buildURL (source = '', reqBase) {
   // issue ref: https://github.com/fastify/fast-proxy/issues/42
-  const cleanSource = source.replace(/\/+/g, '/')
+  let i = 0
+  while (source[i] === '/') {
+    i++
+  }
```

Comment:
```suggestion
  while (source[i] === '/') {
    // Skip leading slash characters in the query parameter value,
    // as they are significant characters of the value.
    i++
  }
```

Reply:
Happy to add this comment, though I wonder if that means we should delete the comment on line 80.

Reply:
Done.

Reply:
Good job with the docblock. I'd still like to see the `while` loop comment so that we don't have to pretend to be a computer to understand what is going on.

Reply:
You got it.